guide/voice/seeking.md

```diff
@@ -0,0 +1,79 @@
+# Seek
+
+Currently, there's no way to seek audio resources with raw discord.js such as:
+
+```js
+const { createAudioResource } = require('@discordjs/voice');
+
+const resource = createAudioResource();
+
+resource.seek(1000); // This will be an error!
+```
```

Comment:
this example seems a bit unnecessary considering it doesn't works?

Reply:
k im gonna delete it